<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ConfigWindow</class>
    <widget class="QDialog" name="ConfigWindow">
        <property name="windowModality">
            <enum>Qt::ApplicationModal</enum>
        </property>
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>480</width>
                <height>270</height>
            </rect>
        </property>
        <property name="minimumSize">
            <size>
                <width>480</width>
                <height>270</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Configuration</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QTabWidget" name="arduinoTabWidget">
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="QWidget" name="arduinoTab">
                        <attribute name="title">
                            <string>Arduino EEPROM Programmer</string>
                        </attribute>
                        <layout class="QFormLayout" name="formLayout">
                            <item row="0" column="0">
                                <widget class="QLabel" name="arduinoSerialPortLabel">
                                    <property name="text">
                                        <string>Serial port</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="0" column="1">
                                <widget class="QComboBox" name="arduinoSerialPortCombo"/>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="arduinoSerialSpeedLabel">
                                    <property name="text">
                                        <string>Speed</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="1">
                                <widget class="QComboBox" name="arduinoSerialSpeedCombo"/>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
            <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="standardButtons">
                        <set>
                            QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Reset
                        </set>
                    </property>
                </widget>
            </item>
        </layout>
    </widget>
    <tabstops>
        <tabstop>arduinoTabWidget</tabstop>
        <tabstop>arduinoSerialPortCombo</tabstop>
        <tabstop>arduinoSerialSpeedCombo</tabstop>
    </tabstops>
    <resources/>
    <connections/>
</ui>
